testing/sdk_security.py

```diff
@@ -145,10 +145,7 @@ def get_default_permissions(
 
 
 def grant_permissions(
-    linux_user: str,
-    role_name: str,
-    service_account_name: str,
-    permissions: List[Dict[str, str]],
+    linux_user: str, role_name: str, service_account_name: str, permissions: List[Dict[str, str]]
```

Comment:
i mean some linters DO complain if there is no trailing comma - but no strong feelings :)

Reply:
I think `black` was ok with this, the PR code checks will fail anyways if its not conformant.